src/fastoad_cs25/models/geometry/geom_components/wing/components/tests/test_wing_geom_comps.py

```diff
@@ -213,6 +214,26 @@ def test_geometry_wing_sweep_inner():
     assert sweep_100_inner == pytest.approx(16.7, abs=1e-1)
 
 
+def test_geometry_compute_center_chord():
+    """Tests computation of the wing center chord and X position"""
+    input_vars = om.IndepVarComp()
+    input_vars.add_output("data:geometry:wing:area", 124.843, units="m**2")
+    input_vars.add_output("data:geometry:wing:kink:chord", 3.985, units="m")
+    input_vars.add_output("data:geometry:wing:kink:y", 6.321, units="m")
+    input_vars.add_output("data:geometry:wing:kink:leading_edge:x:local", 2.275, units="m")
+    input_vars.add_output("data:geometry:wing:root:chord", 6.26, units="m")
+    input_vars.add_output("data:geometry:wing:root:y", 1.96, units="m")
+    input_vars.add_output("data:geometry:wing:tip:chord", 1.882, units="m")
+    input_vars.add_output("data:geometry:wing:tip:y", 15.801, units="m")
+    input_vars.add_output("data:geometry:wing:tip:leading_edge:x:local", 7.222, units="m")
+
+    problem = run_system(ComputeCenterChord(), input_vars)
+    l_center = problem["data:geometry:wing:center:chord"]
```

Comment:
I'm in favour of stopping using this way of fetching variables when those variables have units. Rather I'd recommand using the `get_val` method

```suggestion
    l_center = problem.get_val("data:geometry:wing:center:chord", units="m")
```

Reply:
yes there is only one thing that annoy me that is that get_val with no units gives a array, while with units gives a scalar. i need to open a issue

Reply:
When it is a dimensionless variable I have no issue with using the `problem["..."]` syntax ^^